### DISCUSSION TITLE
Nouvelle visualisation pour graphiques INSEE

### DISCUSSION THREAD

**Discussion by inesdartiguenave** [MTES-MCT/sparte#340] [Idées]
### Situation actuelle 

Graphiques difficilement interprétables. 
Répétition entre graphique avec données brutes de référence puis graphiques avec analyse. 

### Situation souhaitée

Mise en valeur du graphique avec l'indicateur. Sans disperser avec le graphique des données brutes. 

**0/ Pour les valeurs brutes suivantes supprimer le graphique** + associer le tableau de données à l'indicateur mis en perspective 
- surface des territoires; 
- évolution de la pop du territoire; 
- évolution du nb de ménages; 

**1/ graphique surface :** 
- abscisses : conso d’espace;
- ordonnée : surface;
- diagonale du territoire + moyenne française : pourcentage de conso par rapport au territoire ou taux de conso d’espaces par rapport à la surface;

=> intégrer dans le tableau de données un contexte avec les valeurs brutes de surface (tableau supprimé + haut) 

**2/ graphique conso d'espace par nouvel habitant** : 
- abscisses : conso d’espace;
- ordonnée : population;
- diagonale du territoire + moyenne française : conso d'espace par nouvel habitant (en ha)

=> intégrer dans le tableau de données un contexte avec les valeurs brutes de population (tableau supprimé + haut) 

**3/ graphique conso d'espace par nouveau ménage accueilli**: 
- abscisses : conso d’espace;
- ordonnée : ménage;
- diagonale du territoire + moyenne française : conso d'espace par nouveau ménage accueilli 

=> intégrer dans le tableau de données un contexte avec les valeurs brutes de ménage (tableau supprimé + haut) 

**Maquette brouillon** : https://docs.google.com/presentation/d/1D39vlM2nbDAo3v2ZYJPqGV8K7PvH_QT52OG-VFEfWu8/edit?usp=sharing

**Comment by alexisig** [comment]
@inesdartiguenave pour les trois graphiques je vois que tu proposes de montrer des abscisses qui ne sont pas l'année tout en montrant une évolution annuelle. A première vue ça ne semble pas possible. Est-ce que tu crois que tu pourrais faire un graphique entier pour une commune sur le PPT de maquette pour que je comprenne mieux ce que tu souhaiterais ?


![image](https://github.com/MTES-MCT/sparte/assets/16051000/9203a5f7-2b57-4f6f-9efc-e97196a70d24)

**Comment by inesdartiguenave** [comment]
@alexisig oui pour un exemple ppt. Et on a eu une super réunion avec la DDT Haut de France. On va vous partager avec Jérémie les exemples de graphiques / indicateurs à ce sujet

**Comment by alexisig** [comment]
@inesdartiguenave je viens de déplacer le sujet en discussion pour continuer à préparer l'US
